### ISSUE TITLE
cannot install

### ISSUE THREAD

**Issue by DomDumont** [mefechoel/svelte-navigator#90]
When i try to npm install your package I got many errors
because i need is-ci and husky to be present on my machine (they are not) and even after that husky needs a git folder ( my project is not yet under a source control ). It's the first time a package asks for so many prerequisites

**Comment by mefechoel** [comment]
Oh, thanks for the report, that seems to be a problem caused by the latest update. I'll check to see whats happening there. Meanwhile please try installing svelte-navigator@3.1.6.

**Comment by mefechoel** [comment]
I cannot reproduce the error. Are you trying to install the package from git?

**Comment by DomDumont** [comment]
no, just a classic npm install

**Comment by DomDumont** [comment]
the postinstall script seems to be executed on my machine  "postinstall": "is-ci || husky install", I do not know anything about npm

**Comment by mefechoel** [comment]
Yes, that seems to be what is happening, but it should not be able to happen... That script is disabled before the package is published to npm. If you look at the [latest version's package.json (3.3.2)](https://unpkg.com/svelte-navigator@3.2.2/package.json), you'll see that the postinstall script is changed to `_postinstall`. I have no idea whats happening there. I'll need some more information from your setup. What versions of npm and svelte navigator are you using? What are the exact steps to reproduce the error? How is you project set up? Have you tried deleting node_modules and package-lock.json and reinstalling everything again?

**Comment by DomDumont** [comment]
Here are some versions i have :
Windows_NT 10.0.19044
node v16.13.2
npm  v8.18.0
svelte-navigator@3.2.2


deleting node_modules and package_lock doesn't change anything. here is an edited extract of my npm log


164 info run svelte-navigator@3.2.2 postinstall node_modules/svelte-navigator is-ci || husky install
165 http fetch POST 200 871ms
166 timing auditReport:getReport Completed in 1182ms
167 silly audit report {}
168 timing auditReport:init Completed in 0ms
169 timing reify:audit Completed in 1184ms
170 info run svelte-navigator@3.2.2 postinstall { code: 1, signal: null }
171 timing reify:rollback:createSparse Completed in 111ms
172 timing reify:rollback:retireShallow Completed in 0ms
173 timing command:install Completed in 11078ms
174 verbose stack Error: command failed
174 verbose stack     at ChildProcess.<anonymous> (C:\Users\ddumont\AppData\Roaming\npm\node_modules\npm\node_modules\@npmcli\promise-spawn\lib\index.js:63:27)
174 verbose stack     at ChildProcess.emit (node:events:390:28)
174 verbose stack     at maybeClose (node:internal/child_process:1064:16)
174 verbose stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
175 verbose pkgid svelte-navigator@3.2.2
176 verbose cwd D:\Realisations\the-svelte-professor
177 verbose Windows_NT 10.0.19044
178 verbose node v16.13.2
179 verbose npm  v8.18.0
180 error code 1
181 error path D:\test_project\node_modules\svelte-navigator
182 error command failed
183 error command C:\WINDOWS\system32\cmd.exe /d /s /c is-ci || husky install
184 error 'is-ci' n'est pas reconnu en tant que commande interne
184 error ou externe, un programme ex�cutable ou un fichier de commandes.
184 error husky - .git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
185 verbose exit 1
186 timing npm Completed in 11257ms
187 verbose unfinished npm timer reify 1662035465653
188 verbose unfinished npm timer reify:build 1662035476088
189 verbose unfinished npm timer build 1662035476089
190 verbose unfinished npm timer build:deps 1662035476089
191 verbose unfinished npm timer build:run:postinstall 1662035476099
192 verbose unfinished npm timer build:run:postinstall:node_modules/svelte-navigator 1662035476100
193 verbose code 1
194 error A complete log of this run can be found in:

**Comment by DomDumont** [comment]
On another computer I have no problem at all. It may be to my office setup ( we have a  special registry where we get packages ). I will investigate with my IT team and get back to you
